src/Functions/FunctionsConversion.h

```diff
@@ -278,6 +283,17 @@ struct ConvertImpl
                         else
                         {
                             vec_to[i] = static_cast<ToFieldType>(vec_from[i]);
+
+                            if constexpr (is_aarch64 &&
```

Comment:
And actually it affects only one test (I supposed it would be much more), so we can adjust it
https://gh-api.clickhouse.com/play?user=play&url=https%3A%2F%2Fplay-ci.clickhouse.com%2F#U0VMRUNUIGNoZWNrX25hbWUsIHRlc3RfbmFtZSBBUyBuYW1lLCBjb3VudCgpIEFTIGMsIG5hbWUgbm90IGluIChTRUxFQ1QgdGVzdF9uYW1lCkZST00gYGdoLWRhdGFgLmNoZWNrcwpXSEVSRQogICAgY2hlY2tfbmFtZSBMSUtFICclU3RhdGUlJwogICAgQU5EICh0ZXN0X3N0YXR1cyBMSUtFICdGJScgT1IgdGVzdF9zdGF0dXMgTElLRSAnRSUnKQogICAgQU5EIHB1bGxfcmVxdWVzdF9udW1iZXIgPSAzNDYxMgpHUk9VUCBCWSB0ZXN0X25hbWUpIGFzIGZpeGVkXzM0NjEyCkZST00gYGdoLWRhdGFgLmNoZWNrcwpXSEVSRQogICAgY2hlY2tfbmFtZSBMSUtFICclU3RhdGUlJwogICAgQU5EICh0ZXN0X3N0YXR1cyBMSUtFICdGJScgT1IgdGVzdF9zdGF0dXMgTElLRSAnRSUnKQogICAgQU5EIHB1bGxfcmVxdWVzdF9udW1iZXIgPSAwCiAgICBBTkQgY2hlY2tfc3RhcnRfdGltZSA+PSB0b2RheSgpIC0gNwpHUk9VUCBCWSBjaGVja19uYW1lLCBuYW1lCkhBVklORyBtYXgoY2hlY2tfc3RhcnRfdGltZSkgPj0gbm93KCkgLSBJTlRFUlZBTCA0OCBIT1VSIEFORCBjID4gODAKT1JERVIgQlkgZml4ZWRfMzQ2MTIgREVTQywgYyBERVND